pkg/kudoctl/util/kudo/install.go

```diff
@@ -0,0 +1,114 @@
+package kudo
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+)
+
+// Install installs package resources.
+// If skipInstance is set to true, only a package's Operator and OperatorVersion is installed.
+func Install(kc *Client, resources *packages.Resources, skipInstance bool, instanceName, namespace string, parameters map[string]string) error {
```

Comment:
Similar to `UpgradeOperatorVersion` wdyt about?
```suggestion
func InstallOperatorVersion(kc *Client, resources *packages.Resources, skipInstance bool, instanceName, namespace string, parameters map[string]string) error {
```

Reply:
It's doing much more than that, because it also installs an `Operator`

Reply:
`InstallOperator` then?

Reply:
`InstallOperatorAndOperatorVersionAndMaybeAlsoAnInstance` 😆

Reply:
I'd like to avoid "Operator" here, because it's quite overloaded. It could mean the operator as defined by the package, but could also mean the `Operator` the gets deployed on the cluster.

Reply:
`IntallPackage`?